mypy.ini

```diff
@@ -21,3 +21,6 @@ ignore_errors = True
 ;; unless you enjoy writting sth worse than C
 ;; both the establish status of mypy and support from other packages are just wasting your time
 ;; GET AWAY MYPY AND TYPE ANNOTATION !!! WRITTING PYTHON AS IT IS !!!
+
+[tensorcircuit.templates.lattice]
```

Comment:
you cannot add disable error here unless you have sufficient reason, please revert